apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -321,3 +331,55 @@ function CopyCourseInstanceForm({
     </div>
   `;
 }
+
+function CourseInstanceSharing({
+  courseInstance,
+  publicLink,
+}: {
+  courseInstance: CourseInstance;
+  publicLink: string;
+}) {
+  if (!courseInstance.share_source_publicly) {
+    return html`<p>This course instance is not being shared.</p>`;
+  }
+
+  return html`
+    <p>
+      <span class="badge color-green3 me-1">Public source</span>
+      This course instance's source is publicly shared.
+    </p>
+    <div>
+      <label for="publicLink">Public Link</label>
+      <span class="input-group">
+        <input
+          type="text"
+          class="form-control"
+          id="publicLink"
+          name="publicLink"
+          value="${publicLink}"
+          disabled
+        />
```

Comment:
I didn't think to comment on this before today, but: it's suboptimal UX to have the links as text and not, well, links. Specifically, I often look to Cmd-click on a link to open it in a new tab, but I can't do that here: I have to use the "copy" button, open a new tab, and paste it.

I'd be happy to defer this to a separate PR that we can land independently of this one, but I think this would be a nice improvement (both here and across all of these links on the settings pages).

Reply:
Totally agree about the links not being links.

I'll make an issue and we can do a separate PR that cleans up/dedupes and improves these.

Reply:
new issue for this here: #11983 feel free to add any more detail there you tihnk is important